README.md

```diff
@@ -31,6 +31,14 @@ docker-compose up --build
 
 ### With Bundler:
 
+Install bundler first:
+
+```
+sudo gem install bundler
```

Comment:
Don’t you need to install both the `bundler` and `jekyll`?

Reply:
(maybe `jekyll` is install when you `bundle install`?)

Reply:
No idea. To my memory, I didn't use `jekyll` before, so it was likely not installed on my machine. I suppose installing only `bundler` did the trick for me.